tests/tests/operation_tests.cpp

```diff
@@ -7423,7 +7424,450 @@ BOOST_FIXTURE_TEST_SUITE(auction_window_tests, votes_extended_fixture)
+            BOOST_CHECK_EQUAL(abs(auw_tokens - total_comment_fund.reward_fund().amount.value) < allowed_tokens_delta, true);
+            BOOST_CHECK_EQUAL(fabs(0.25 - modeled_voters_reward_percent) < allowed_percent_delta, true);
+        }
+        FC_LOG_AND_RETHROW()
+    }
+
+    BOOST_AUTO_TEST_CASE(auction_window_tokens_to_reward_fund_empty_case) {
+        try {
+            BOOST_TEST_MESSAGE("Auction window reward goes to reward fund.");
+
+            // Needed to be sured, that auction window's been already enabled.
+            generate_blocks(fc::time_point_sec(STEEMIT_HARDFORK_0_6_REVERSE_AUCTION_TIME), true);
+
+            auto &wso = db->get_witness_schedule_object();
+            db->modify(wso, [&](witness_schedule_object &w) {
+                w.median_props.auction_window_size = 10 * 60;
+            });
+
+            generate_block();
+
+            ACTOR(alice);
+            int voters_count = 10;
+
+            comment_operation comment;
+            comment_options_operation cop;
+            signed_transaction tx;
+            set_price_feed(price(ASSET("1.000 GOLOS"), ASSET("1.000 GBG")));
+
+            BOOST_TEST_MESSAGE("Create a post.");
+            comment.author = "alice";
+            comment.permlink = "testauw2";
+            comment.parent_permlink = "test";
+            comment.title = "test";
+            comment.body = "Let's test auction window improvements!";
+
+            tx.operations.clear();
+            tx.signatures.clear();
+            GOLOS_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, comment));
+            generate_block();
+
+            protocol::comment_auction_window_reward_destination dest;
+            dest.destination = protocol::to_reward_fund;
+
+            cop.author = comment.author;
+            cop.permlink = comment.permlink;
+
+            cop.extensions.insert(dest);
+
+            tx.operations.clear();
+            tx.signatures.clear();
+
+            GOLOS_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, cop));
+            generate_block();
+            auto & alice_post = db->get<comment_object, by_permlink>(boost::make_tuple(comment.author, comment.permlink));
+            generate_voters(voters_count);
+
+            BOOST_TEST_MESSAGE("Create votes.");
+            BOOST_CHECK_EQUAL(alice_post.cashout_time, alice_post.created + STEEMIT_CASHOUT_WINDOW_SECONDS);
+
+            generate_blocks((alice_post.created + alice_post.auction_window_size), true);
+            vote_sequence(comment.author, comment.permlink, voters_count, 5);
+
+
+            comment_fund total_comment_fund(*db);
+            comment_reward alice_post_reward(*db, total_comment_fund, alice_post);
+            auto& gpo = db->get_dynamic_global_properties();
+
+            generate_blocks((alice_post.cashout_time), true);
+
+            share_type rwd = 0;
+            for (int i = 0; i < voters_count; i++) {
+                std::string acc_name = "voter" + std::to_string(i);
```

Comment:
Excessive variable acc_name.

Reply:
Why? It's okay to write like that.

Reply:
@AKorpusenko It's
ok
ay
to
wr
it
el
ik
ew
ha
t.
An
dv
er
ys
im
pl
et
or
ea
ds
uc
hw
ri
ti
ng
s!
Understood?